pyftpdlib/log.py

```diff
@@ -146,14 +147,28 @@ def is_logging_configured():
 
 
 # TODO: write tests
+
 def config_logging(level=LEVEL, prefix=PREFIX, other_loggers=None):
-    # Little speed up
-    if "(process)" not in prefix:
+    # Speedup logging by preventing certain internal log record info to
+    # be unnecessarily fetched. This results in about 28% speedup. See:
+    # * https://docs.python.org/3/howto/logging.html#optimization
+    # * https://docs.python.org/3/library/logging.html#logrecord-attributes
+    # * https://stackoverflow.com/a/38924153/376587
+    key_names = re.findall(
```

Comment:
`re.findall` returns a list - with the amount of times that you do `not in key_names` in the lines below here, I wonder if it'd be worth converting `key_names` to a set first?

Reply:
Yes makes sense. Done in dfd6891. Thanks Andrew.